web/src/components/watches/DownstreamWatchVersionDiff.jsx

```diff
@@ -47,12 +47,6 @@ class DownstreamWatchVersionDiff extends React.Component {
         } else {
           this.setState({ secondApplicationTree: files });
         }
-        if (
-          this.state.firstApplicationTree.files &&
-          this.state.secondApplicationTree.files
-        ) {
-          this.setState({ loadingFileTrees: false });
-        }
```

Comment:
My guess is that this is checking to see if all of the data is loaded before removing the loading state. `setState` is async though so I'm not sure how this ever worked.